x-pack/solutions/security/plugins/security_solution/public/common/lib/telemetry/events/telemetry_events.ts

```diff
@@ -18,7 +17,6 @@ import { previewRuleTelemetryEvents } from './preview_rule';
 import { siemMigrationsTelemetryEvents } from './siem_migrations';
 
 export const telemetryEvents = [
-  ...assistantTelemetryEvents,
```

Comment:
Note: this now registers through: https://github.com/elastic/kibana/blob/af89e6c7bcc13c920026528c080a78c94ce9c0f9/x-pack/solutions/security/plugins/elastic_assistant/public/src/common/lib/telemetry/telemetry_service.ts#L31-L33

I still need to verify telemetry is operational -- last I checked in the [staging telemetry dashboard](https://telemetry-v2-staging.elastic.dev/s/securitysolution/app/dashboards#/view/507bf390-2a68-11ee-adde-d5df298171dd) I didn't see anything for the cluster_uuid's that were deployed as part of the build, but not 100%, so need to confirm with local deployment with telemetry enabled.